test/jdk/java/awt/event/KeyEvent/KeyChar/KeyCharTest.java

```diff
@@ -45,7 +50,6 @@
         Locale.setDefault(Locale.ENGLISH);
 
         Toolkit.getDefaultToolkit().addAWTEventListener(new AWTEventListener() {
-
```

Comment:
This test was excluded on macOS because it leaves some buttons pressed. I think the reason was an exception on EDT, then the jtreg stops the test when we get an exception below but before we release the delete button on the main thread.